app/client/views/application/applicationCtrl.js

```diff
@@ -77,7 +78,11 @@ angular.module('reg')
               type: "success",
               confirmButtonColor: "#e76482"
             }, function(){
-              $state.go('app.dashboard');
+              if(firstTime){
+                $state.go('app.dashboard');
```

Comment:
Here's where I'd like to make the whole site reload so that everything will appear in the sidebar.
Otherwise the user will have to reload manually which is very confusing if you don't know this.